powermeter/vzlogger.py

```
@@ -6,12 +6,18 @@ class VZLogger(Powermeter):
     def __init__(self, ip: str, port: str, uuid: str):
         self.ip = ip
         self.port = port
-        self.uuid = uuid
+        self.uuids = dict(zip([u.strip() for u in uuid.split(",")], [0,1,2]))
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)